### ISSUE TITLE
Strange Setter detector in Slitherin doesn't function as expected - Throws False Positives

### ISSUE THREAD

**Issue by zaryab2000** [pessimistic-io/slitherin#147]
I was running the slitherin tool in a smart contract that had the following constructor 👇
```    
constructor(
        string memory name,
        string memory symbol,
        uint256 totalSupply,
        Fees memory fees_,
        address routerAddress,
        address routerAddress2
    ) ERC20(name, symbol) Ownable(msg.sender) {
        fees = fees_;

        uniswapV2Router = IUniswapV2Router02(routerAddress);
        uniswapV2Router2 = IUniswapV2Router02(routerAddress2);
        uniswapV2Pair = IUniswapV2Factory(uniswapV2Router.factory()).createPair(
            address(this),
            uniswapV2Router.WETH()
        );
        uniswapV2Pair2 = IUniswapV2Factory(uniswapV2Router2.factory())
            .createPair(address(this), uniswapV2Router2.WETH());

        _isExcludedFromFee[BURN_ADDRESS] = true;
        _isExcludedFromFee[address(this)] = true;
        _isExcludedFromFee[owner()] = true;

        _mint(msg.sender, totalSupply);
    }
```

---
Slither threw an info-detector that says:

> Function Ownable.constructor(address) (flat.sol#491-496) is a strange setter. Nothing is set in constructor or set in a function without using function parameters

This is incorrect as it can clearly be seen that :

- There are a few state variables that are being updated inside the constructor
- Additionally, the arguments passed are also being used in the constructor.

**Comment by olegggatttor** [comment]
I'll take it!

**Comment by ndkirillov** [comment]
Released with #157
